package.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "github-repo-tracker",
+    "version": "1.0.0",
+    "description": "[React State and Eventing](https://github.com/getfutureproof/fp_guides_wiki/wiki/React-State-and-Eventing)",
+    "main": "index.js",
+    "scripts": {
+        "test": "jest --watch --setupFilesAfterEnv ./src/test/setupTests.js",
+        "coverage": "jest --setupFilesAfterEnv ./src/test/setupTests.js --coverage --watchAll=false",
+        "dev": "webpack-cli serve --mode development --config config/webpack.config.dev.js",
+        "build": "webpack --config config/webpack.config.production.js"
+    },
+    "jest": {
+        "moduleNameMapper": {
+            "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
+            "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
+        }
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/getfutureproof/fp_study_notes_react_state_and_eventing.git"
```

Comment:
hmmmmmmm interesting..... I wonder where this is from

(I dot expect you to re-write this while thing because we didn't either but just make sure that you get rid of any trace its not yours lol)